### PR TITLE
Artwork `aggregateOffer` metadata

### PR DESCRIPTION
**Pull request by sepans**
~HOLD: Depends on #3015~

Fixes [PURCHASE-1421](https://artsyproduct.atlassian.net/browse/PURCHASE-1421)

Also fixes the price format `string` -> `number`

I basically copied the code from `ArtistMeda` [here](https://github.com/artsy/reaction/blob/master/src/Apps/Artist/Components/ArtistMeta.tsx#L54).

There is opportunity to merge the logic shared between the `Product` tag on these two pages and also the collection page and make the code dryer but since there are some open questions(especially about collection products) I will create a ticket to come back and refactor. Ideally we want to move the query fragment under the `Product` [component](https://github.com/sepans/reaction/blob/c4067a374237862a46ddaafbadf2ab46df70c16c/src/Components/v2/Seo/Product.tsx#L19) to move the logic in a single place.

Before:
```JSON
{
  "@context": "http://schema.org",
  "@type": "Product",
  "name": "Dabeiyuzhou | 2nd Genome Test Partner | artwirj (2000) | Available for Sale | Artsy",
  "image": "...",
  "description": "Available for sale from Invoicing Demo Partner, Dabeiyuzhou | 2nd Genome Test Partner, artwirj (2000), Oil, 20 × 30 in",
  "url": "https://staging.artsy.net/artwork/dabeiyuzhou-2nd-genome-test-partner-artwirj",
  "width": "20 in",
  "height": "30 in",
  "brand": {
    "@type": "Person",
    "name": "Dabeiyuzhou | 2nd Genome Test Partner"
  },
  "category": "Print",
  "productionDate": "2000",
  "offers": {
    "@type": "Offer",
    "priceSpecification": {
      "@type": "PriceSpecification",
      "priceCurrency": "USD",
      "minPrice": "1,600",
      "maxPrice": "10,000"
    },
    "availability": "https://schema.org/InStock",
    "seller": {
      "@type": "ArtGallery",
      "name": "Invoicing Demo Partner"
    }
  }
}
```

After:
```JSON
{
   "@context":"http://schema.org",
   "@type":"Product",
   "name":"Dabeiyuzhou | 2nd Genome Test Partner | artwirj (2000) | Available for Sale | Artsy",
   "image":"...",
   "description":"Available for sale from Invoicing Demo Partner, Dabeiyuzhou | 2nd Genome Test Partner, artwirj (2000), Oil, 20 × 30 in",
   "url":"undefined/artwork/dabeiyuzhou-2nd-genome-test-partner-artwirj",
   "width":"20 in",
   "height":"30 in",
   "brand":{
      "@type":"Person",
      "name":"Dabeiyuzhou | 2nd Genome Test Partner"
   },
   "category":"Print",
   "productionDate":"2000",
   "offers":{
      "@type":"AggregateOffer",
      "lowPrice":1600,
      "highPrice":10000,
      "priceCurrency":"USD",
      "availability":"https://schema.org/InStock",
      "seller":{
         "@type":"ArtGallery",
         "name":"Invoicing Demo Partner"
      }
   }
}
```

### CODE REVIEW

**Comment by sepans** [comment]
ticket for code cleanup: https://artsyproduct.atlassian.net/browse/PURCHASE-1667

File: src/Apps/Artwork/Components/Seo/SeoDataForArtwork.tsx [hunk 1]
```diff
@@ -124,53 +122,39 @@ export const SeoDataForArtworkFragmentContainer = createFragmentContainer(
...
-    priceCurrency: price_currency,
-  }
-}
-
-const splitPriceRange = (priceRange: string) => {
-  const minAndMaxPrice = priceRange.split("-")
-  return {
-    minPrice: trim(minAndMaxPrice[0]).replace("$", ""),
-    maxPrice: trim(minAndMaxPrice[1]),
+export const offerAttributes = (artwork: SeoDataForArtwork_artwork) => {
```
Comment (sepans) [inline src/Apps/Artwork/Components/Seo/SeoDataForArtwork.tsx]: This function I copied from artist page without changing. I created a ticket to revisit to either put this along with query fragment in `<Product>` component or just reuse this function from a util file.

File: src/Apps/Artwork/Components/Seo/__tests__/SeoDataForArtwork.fixture.ts [hunk 2]
```diff
@@ -7,8 +7,6 @@ export const SeoDataForArtworkFixture: SeoDataForArtwork_Test_QueryRawResponse["
   is_price_hidden: false,
   is_price_range: false,
   listPrice: null,
-  price_currency: "USD",
```
Comment (sepans) [inline src/Apps/Artwork/Components/Seo/__tests__/SeoDataForArtwork.fixture.ts]: currency now comes from `listPrice`

**Comment by artsyit** [comment]
:rocket: PR was released in v22.2.3 :rocket:

**Pull request merged**